spec/controllers/api/v1/notes_controller_spec.rb

```diff
@@ -0,0 +1,134 @@
+require 'rails_helper'
+
+shared_examples 'successfully response' do
```

Comment:
```suggestion
shared_examples 'successful response' do
```
Los nombres de los shared_examples siempre tienen que tener sentido luego de un it_behaves_like

Reply:
Cambie el nombre para que tenga mas sentido